.github/workflows/heroku-pull-request.yml

```diff
@@ -61,8 +61,16 @@ jobs:
         run: heroku addons:create heroku-postgresql:hobby-dev --app ${{ env.HEROKU_APP_NAME }}
 
       - name: Add Heroku remote
+        continue-on-error: true
+        id: add-remote
         run: heroku git:remote --app=${{ env.HEROKU_APP_NAME }}
```

Comment:
`for i in $(range 10); do { heroku git:remote --app=${{ env.HEROKU_APP_NAME }} ; exit 0 } ; sleep 1; done`